tests/unit/test_charm.py

```diff
@@ -399,8 +344,36 @@ def test_db_relation(self):
             "database name should be set after relation joined",
         )
 
+    def test_add_admin_user(self):
```

Comment:
We have a standard for the test docstring, is it applicable here? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-structure